metadata-ingestion/src/datahub/ingestion/source/unity/source.py

```diff
@@ -309,9 +309,19 @@ def get_workunits_internal(self) -> Iterable[MetadataWorkUnit]:
             )
 
         if self.config.is_profiling_enabled():
-            self.report.report_ingestion_stage_start("Wait on warehouse")
-            assert wait_on_warehouse
-            wait_on_warehouse.result()
+            self.report.report_ingestion_stage_start("Start warehouse")
+            # Can take several minutes, so start now and wait later
```

Comment:
This comment is not relevant here. May add a comment how the warehouse started earlier may turn off automatically by the time code reaches here and so we need to start warehouse again.

Reply:
will update the comment